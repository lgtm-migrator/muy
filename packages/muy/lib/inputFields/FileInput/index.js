import React from "react"
import PropTypes from "prop-types"

const FileInput = ({ accept, id, name, ...props }) => (
  <input accept={accept} id={id} name={name} type="file" {...props} />
)

export default FileInput

FileInput.defaultProps = {
  id: "file",
  name: "file",
}

FileInput.propTypes = {
  accept: PropTypes.string,
  id: PropTypes.string,
  name: PropTypes.string,
}
