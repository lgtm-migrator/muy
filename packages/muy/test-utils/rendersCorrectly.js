import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
