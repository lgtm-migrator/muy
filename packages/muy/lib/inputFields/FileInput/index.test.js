import React from "react"
import rendersCorrectly from "../../../test-utils/rendersCorrectly"
import FileInput from "."

describe("<FileInput /> field", () => {
  it("should render correctly", () => {
    rendersCorrectly(<FileInput />)
  })
})
