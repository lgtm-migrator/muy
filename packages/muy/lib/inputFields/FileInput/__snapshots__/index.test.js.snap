// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileInput /> field should render correctly 1`] = `
<input
  id="file"
  name="file"
  type="file"
/>
`;
